builtin/credential/okta/cli.go

```diff
@@ -38,13 +38,8 @@ func (h *CLIHandler) Auth(c *api.Client, m map[string]string) (*api.Secret, erro
 		"password": password,
 	}
 
-	mfa_method, ok := m["method"]
```

Comment:
Were these use copy-pasted values that we're removing?

Reply:
I thought so but after looking deeper I see they could be used by the legacy MFA system. I've restored them.